firstvoices/backend/serializers/widget_serializers.py

```diff
@@ -0,0 +1,38 @@
+from rest_framework import serializers
+
+from backend.models.widget import SiteWidget, Widget, WidgetSettings
+from backend.serializers.base_serializers import SiteContentLinkedTitleSerializer
+from backend.serializers.fields import SiteHyperlinkedIdentityField
+
+
+class WidgetSettingsSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = WidgetSettings
+        fields = ("key", "value")
+
+
+class WidgetDetailSerializer(serializers.ModelSerializer):
+    url = serializers.HyperlinkedIdentityField(view_name="api:widget-detail")
+    widget_type = serializers.CharField()
+    format = serializers.CharField(source="get_format_display")
+    settings = WidgetSettingsSerializer(source="widgetsettings_set", many=True)
+
+    class Meta:
+        model = Widget
+        fields = ("url", "id", "title", "widget_type", "format", "settings")
```

Comment:
I think the json field could be "type" instead of "widgetType", to match the old data format